configure.ac

```diff
@@ -2659,6 +2658,16 @@ vici socket path        : ${vici_socket}
 The above user and group must have read/write access to the state file
 directory and to the config files in the config file directory."
 
+if test -n "$enable_datacenter"; then
```

Comment:
can we update installation.rst to note that these three configure options are deprecated now?